### ISSUE TITLE
bigip_facts:  get_fw_rule NotImplemented Error

### ISSUE THREAD

**Issue by lndevnull** [ansible/ansible-modules-extras#109]
Getting the following error after a recent F5 upgrade to 11.6 from 11.5

```
failed: [localhost -> 127.0.0.1] => {"failed": true}
msg: received exception: Server raised fault: 'Exception caught in LocalLB::urn:iControl:LocalLB/VirtualServer::get_fw_rule()
Common::NotImplemented
```

Role is defined as the following: 

```
  tasks:
  - name: Get F5 Virtual Servers
    local_action: >
      bigip_facts
      server=esilo-f5-corp
      user={{ f5_user }}
      password={{ f5_passwd }}
      include=virtual_server
    register: bigip_vars
```

Ansible 1.7.2

**Comment by lndevnull** [comment]
Temporary fix was to comment out both get_fw_rule definitions.

**Comment by srgvg** [comment]
ping @mhite

**Comment by mhite** [comment]
Haven't moved past 11.5 yet but will grab the 11.6 VM and see what I can figure out.

**Comment by mhite** [comment]
From iControl documentation:

```
get_fw_rule *IMPORTANT* This method has been deprecated immediately. As of v11.6.0, attempting to perform this method will return a not-implemented exception. Gets the firewall rules for the specified virtual servers.
```

Removal seems the best course of action.

**Comment by mhite** [comment]
Feel free to give this a whirl:

https://github.com/ansible/ansible-modules-extras/pull/272

**Comment by mhite** [comment]
Or #273, which is a bit more of a general fix which should help if F5 does this to me again. ;)

**Comment by bcoca** [comment]
closing vea merged #273
